### ISSUE TITLE
NEW BUG 

### ISSUE THREAD

**Issue by rectrick** [cgtinker/BlendArMocap#171]
https://prnt.sc/warhPjZeSxz1

**Comment by mak448a** [comment]
I don't think this is a bug. You need to click the "install dependencies" button in the addons tab of blender settings.
